content/communities/government-contact-center-council.md

```diff
@@ -1,53 +1,34 @@
 ---
-# View this page at https://digital.gov/communities/government-contact-center-council
-# Learn how to edit our pages at https://workflow.digital.gov
-
 slug: government-contact-center-council
 date: 2013-11-21 3:28:02 -0400
```

Comment:
This should have the CoP lead info added

Reply:
PR for adding Tim as an author: https://github.com/GSA/digitalgov.gov/pull/3619